src/components/Person/Person.jsx

```diff
@@ -1 +1,22 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  return (
+    <div>
+      <section className="Person">
+        <h2 className="Person__name">My name is {person.name}</h2>
+        {person.age ? <p className="Person__age">I am {person.age}</p> : null}
+
+        {person.isMarried && (
+          <p className="Person__partner">
+            {person.sex === 'm'
+              ? `${person.partnerName} is my wife`
+              : `${person.partnerName} is my husband`}
+          </p>
+        )}
+
+        <p className="Person__partner">
```

Comment:
Camille, I believe the test `{!person.isMarried && ...` should come before this line, so it prevents the rendering for the whole <p> tag instead of only the text

Reply:
ah sim, vou fazer esse ajuste e fazer o deploy pra você conseguir testar! Eu sempre esqueço :/

Reply:
Pedro, eu até tentei fazer o deploy mas fiquei tentando rodar os testes ou dar npm start mas fiquei recebendo isso:

/bin/sh: /Users/camillehaus/Documents/Mate_Academy/react_person/node_modules/.bin/vite: No such file or directory

A unica coisa q funcionou foi o commit.. Fiz a alteração que você recomendou

Reply:
Provavelmente deve ter que instalar de novo o projeto, por causa das mudanças nos scripts de teste da Mate. No problem